src/components/admin/WorldUserCard/WorldUserCard.tsx

```diff
@@ -16,6 +19,8 @@ export interface WorldUserCardProps {
   userId?: UserId;
 }
 
+const DEFAULT_AVATAR = DEFAULT_AVATAR_LIST?.[0];
```

Comment:
[Question] I think think won't ever be undefined, unless we deleted it
So we can rewrite it like `const [DEFAULT_AVATAR] = DEFAULT_AVATAR_LIST;`